generators/ionic/resources/oauth2/src/app/auth/auth-config.service.ts

```diff
@@ -19,9 +19,8 @@ export class AuthConfigService {
         if (this.authConfig.issuer.endsWith('/')) {
           this.authConfig.issuer = this.authConfig.issuer.substring(0, this.authConfig.issuer.length - 1);
         }
-        // Override issuer and client ID with values from API
+        // Override issuer with value from API. Client ID is configured in environment.
         environment.oidcConfig.server_host = this.authConfig.issuer;
```

Comment:
Why get the issuer but not the clientId?

Reply:
When we first implemented it with Keycloak, we didn't know that the same client ID wouldn't work with Ionic. JHipster's OAuth flow requires a client secret. With Ionic, you shouldn't have secrets on the client so PKCE is used instead. We might be able to use the server-side settings just for the tests, but then you'd have to add `http://localhost:8100` as allowed redirects/origins in the JHipster OIDC app's configuration.

Reply:
Ok, so clientId differs.